src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidget.java

```diff
@@ -1,23 +1,29 @@
 package org.sagebionetworks.web.client.widget.discussion;
 
 import org.sagebionetworks.repo.model.discussion.DiscussionThreadBundle;
+import org.sagebionetworks.web.client.GWTWrapper;
 import org.sagebionetworks.web.client.PortalGinInjector;
 
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
-public class ThreadWidget implements ThreadWidgetView.Presenter{
+public class DiscussionThreadWidget implements DiscussionThreadWidgetView.Presenter{
 
-	ThreadWidgetView view;
+	DiscussionThreadWidgetView view;
 	PortalGinInjector ginInjector;
+	GWTWrapper gwtWrapper;
+	boolean areRepliesConfigure;
```

Comment:
I don't understand why areRepliesConfigure exists, and assume that this will go away as implementation progresses.